attestation-service/verifier/src/az_snp_vtpm/mod.rs

```diff
@@ -48,9 +48,10 @@ impl Verifier for AzSnpVtpm {
     /// The following verification steps are performed:
     /// 1. TPM Quote has been signed by AK included in the HCL variable data
     /// 2. Attestation report_data matches TPM Quote nonce
-    /// 3. SNP report's report_data field matches hashed HCL variable data
-    /// 4. SNP Report is genuine
-    /// 5. SNP Report has been issued in VMPL 0
+    /// 3. TPM Quote's PCRs' digest matches the PCR digest from attestation report
```

Comment:
there is no PCR digest in the attestation report, the digest ist in the TPM Quote.

```suggestion
    /// 3. TPM PCRs' digest matches the digest in the Quote
```